Add Logger to Output Query Logs (#4)
DeNA/mysql_rewinder: README.md

If you want to use MysqlRewinder with ActiveRecord, do the following:

* Generate db_configs from `ActiveRecord::Base.configurations`

> * Pass `ActiveRecord::SchemaMigration.new(nil).table_name` and `ActiveRecord::Base.internal_metadata_table_name` to `MysqlRewinder.setup` as `except_tables`

Reviewer: Thanks!

MysqlRewinder.setup(db_configs, except_tables: except_tables)
```

### Logging

> If you want to enable logging, do the following:
> 
> * Generate db_configs from `ActiveRecord::Base.configurations`
> * Pass `logger: Logger` to `MysqlRewinder.setup`.
> 
> ```ruby
> MysqlRewinder.setup(db_configs, logger: Logger.new(STDOUT))
> ```

Reviewer: I think we can simplify like:

```suggestion
If you want to enable logging, specify `logger` for `MysqlRewinder.setup`

```ruby
MysqlRewinder.setup(db_configs, logger: Logger.new(STDOUT))
```
```
